subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -1048,3 +1048,40 @@ Alternatively, for Kotlin implementations, the Kotlin compiler can be configured
 include::sample[dir="snippets/configurationCache/samConversion/groovy",files="build.gradle[tags=configure]"]
 include::sample[dir="snippets/configurationCache/samConversion/kotlin",files="build.gradle.kts[tags=configure]"]
 ====
+
+[[config_cache:not_yet_implemented:accessing_top_level_at_execution]]
+=== Accessing top-level methods and variables at execution time
```

Comment:
📝 I think we should include the word _script_ somewhere here since the context is not clear like in the other page. 

_Accessing top-level script methods and variable at execution time_, perhaps?